presto-hive/src/main/java/com/facebook/presto/hive/HiveMetadata.java

```diff
@@ -1723,6 +1740,90 @@ public boolean supportsMetadataDelete(ConnectorSession session, ConnectorTableHa
         return true;
     }
 
+    @Override
+    public boolean isPushdownFilterSupported(ConnectorSession session, ConnectorTableHandle tableHandle)
```

Comment:
Can we add a unit tests or integration tests for this code path?

Reply:
@highker This path is not fully implemented yet. Only the planner portion is available. Hence, an end-to-end queries don't work yet.